php-frontend/src/main/java/org/sonar/php/parser/NewPHPGrammar.java

```diff
@@ -50,6 +58,104 @@ public ExpressionTree EXPRESSION() {
    * [ START ] Expression
    */
 
+  public LiteralTree COMMON_SCALAR() {
+    return b.<LiteralTree>nonterminal(PHPLexicalGrammar.COMMON_SCALAR)
+      .is(b.firstOf(
+        f.heredocLiteral(b.token(PHPLexicalGrammar.HEREDOC)),
+        NUMERIC_LITERAL(),
+        // FIXME: when ENCAPS_LIST is migrated
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)